src/rocksdb.rs

```diff
@@ -975,6 +1096,36 @@ fn writebatch_works() {
     let opts = Options::default();
     assert!(DB::destroy(&opts, path).is_ok());
 }
+#[test]
+fn multi_get_test() {
+    let path = "_rust_rocksdb_multi_get_test";
+    {
+        let db = DB::open_default(path).unwrap();
+        let p = db.put(b"mk1", b"v1111");
+        assert!(p.is_ok());
+        let p = db.put(b"mk2", b"v2222");
+        assert!(p.is_ok());
+        let p = db.put(b"mk3", b"v3333");
+        assert!(p.is_ok());
+
+        let all = db.multi_get(vec![b"mk1",b"mk2",b"mk3"]);
+        assert_eq!(all.len(), 3);
+        match all[0] {
+            Ok(Some(ref u)) => assert_eq!(u.to_utf8(),Some("v1111")),
+            _ =>  assert!(false)
+        }
+        match all[1] {
+            Ok(Some(ref u)) => assert_eq!(u.to_utf8(),Some("v2222")),
+            _ =>  assert!(false)
+        }
+        match all[2] {
+            Ok(Some(ref u)) => assert_eq!(u.to_utf8(),Some("v3333")),
+            _ =>  assert!(false)
+        }
+    }
+    let opts = Options::default();
+    assert!(DB::destroy(&opts, path).is_ok());
+}
```

Comment:
Can we add a test here that requests a key that is not present, and makes sure the 1:1 mapping between requested keys and the resulting `Result`?